src/screens/admin/ComMemberAdmin.jsx

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import DashboardLayout from '../../layout/dashboardLayout/DashboardLayout'
+import { configFunc, getApi } from '../../utils/apiFunc'
+import './ComMemberAdmin.scss';
+
+const ComMemberAdmin = () => {
+    const [data, setData] = useState([]);
+    const [success, setSuccess] = useState(null);
+    // fetching current community
+    const currentCommunity = localStorage.getItem('currentCommunity')
+    ? JSON.parse(localStorage.getItem('currentCommunity'))
+    : null
+    const dispatch = useDispatch();
+    const config = configFunc()
+
+    useEffect(() => {
+        getMemberDetails()
+    }, [success])
+
+    const getMemberDetails = async () => {
+        const { data } = await getApi(
+        dispatch,
+        `${process.env.REACT_APP_API_BASE_URL}/api/communities-users/community/${currentCommunity.id}/details`
+        )
+        setData(data.results)
+    }
+
+    const allowAccess = async (id) => {
+        const {data} = await axios.put(`${process.env.REACT_APP_API_BASE_URL}/api/communities-users/${id}/community/${currentCommunity.id}`,
+         {role: "manager"}, config)
+         setSuccess(data)
+    }
+
+    return (
+        <DashboardLayout title={currentCommunity.name}>
+            <div className="com-member-container">
+                <div className="com-member-header">
+                {
+                    ["firstName", "lastName", "email", "phone", "dateOfBirth", "role", "options"].map(el => {
```

Comment:
this is not using table component created for admin panel

Reply:
Table component is in another branch(admin-category-page), so it will be replaced after the branch is merged.